rootfs-builder/suse/config.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<image schemaversion="6.8" name="openSUSE-Leap-Kata" displayname="openSUSE Leap Kata">
+    <description type="system">
+        <author>openSUSE Project</author>
+        <contact>opensuse-factory@opensuse.org</contact>
+        <specification>openSUSE Leap Kata</specification>
+    </description>
+    <preferences>
+        <version>1.0.0</version>
```

Comment:
OOI, what is this version for?

Reply:
that's a way to track the version of the config file itself